WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -161,6 +185,138 @@ extension PointOfSaleAggregateModel {
             await cardPresentPaymentService.disconnectReader()
         }
     }
+
+    /// Starts a payment immediately if a reader is connected.
+    /// Otherwise, schedules a payment to start the next time a reader connects.
+    /// Note that any schedlued payments are cancelled by `cancelReaderPreparation`
+    /// e.g. when the TotalsView goes offscreen.
+    func startPaymentWhenCardReaderConnected() async {
```

Comment:
👀 outside of the scope of this PR, just noting here that this most likely something we'll need to change soon as part of allowing cash payments.

Reply:
I don't think we will neccesarily – it might just need to be renamed to `prepareReaderWhenConnected`.

Having the reader prepped for payment doesn't mean someone needs to tap their card – if the merchant selects cash, we can cancel the payment intent.